src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -635,7 +609,8 @@ def sizing(self, parts=None, size=None):
         vg_free_count = util.str_to_int(self.vg_free_count)
 
         if size:
-            extents = int(size * vg_free_count / self.free)
+            size= size*1024*1024*1024
```

Comment:
Yes that name would make sense. But since it is a keyword arg this would require changing all call sites too. Since there will be a follow-up PR to get the sizing byte-based anyway, I'd prefer to leave it as is for now.